### ISSUE TITLE
Window control is missing in ubuntu mate

### ISSUE THREAD

**Issue by irfansyah11** [vinceliuice/Mojave-gtk-theme#103]
![Screenshot at 2020-02-02 01-18-39](https://user-images.githubusercontent.com/58948234/73597153-b2a94680-455b-11ea-8483-5f3b793f0350.png)

**Comment by vinceliuice** [comment]
Could you tell me your Firefox version?

**Comment by irfansyah11** [comment]
I use version 72.0.2 (64-bit)

**Comment by vinceliuice** [comment]
OK! I'll try fix this

**Comment by irfansyah11** [comment]
Thank you, cheer up. Good luck.
I wait for your work: D

**Comment by vinceliuice** [comment]
Did you use any plugin to move titlebutton on panel?

**Comment by vinceliuice** [comment]
![2020-02-02 14-05-04 的屏幕截图](https://user-images.githubusercontent.com/7604295/73604351-60ebd500-45ca-11ea-8ea3-38e16f572281.png)

**Comment by vinceliuice** [comment]
I got no issue about titlebutton

**Comment by irfansyah11** [comment]
no. I am not fiddling with the window control button. I just moved it from right to left in Mate Tweak.

**Comment by irfansyah11** [comment]
Solved. Apparently the problem in the Global menu is not yet stable.
When I delete the global menu items in the top panel, Windows Control reappears.

**Comment by trmdi** [comment]
I've still not understood the root cause, but you can use this workaround:
```CSS
 #toolbar-menubar[autohide="true"]:not([inactive]) + #TabsToolbar > .titlebar-buttonbox-container {
     visibility: visible !important;
 }
```
Not sure if it causes any other bug.
Probably it relates to this: https://dxr.mozilla.org/mozilla-release/source/browser/base/content/browser.css#329

**Comment by trmdi** [comment]
@vinceliuice The issue appears when you're using Firefox with the global menu in any DE.
